x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -668,6 +669,25 @@ function addRACRuleTypes(
     : doc;
 }
 
+function addThreatIndicatorPathToThreatMatchRules(
+  doc: SavedObjectUnsanitizedDoc<RawAlert>
+): SavedObjectUnsanitizedDoc<RawAlert> {
+  const isThreatMatchRule = doc.attributes.params.type === 'threat_match';
```

Comment:
Should we also check for `isSiemSignalsRuleType(doc)` here?

Can we handle the possibility that `doc.attributes.params` is `null` or `undefined`? Not sure if this could be the case for any rule type, but probably safer to do a null check?